### PR TITLE
39 tabs refactor

### PR DESCRIPTION
**Pull request by stuartromanek**
splits tabs into Tab and Body components and stories them together in an AposModal. Mixin provides tab switching communication between tabs and panes

### CODE REVIEW

File: src/components/modalTabs/AposModalTabs.stories.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            fields: [],
+            uid: (Math.floor(Math.random() * Math.floor(10000)))
+          }
+        ],
+        breadcrumbs
+      };
+    },
+    template: `
+      <div>
+        <button type="button" class="apos-button">
```
Comment (abea) [inline src/components/modalTabs/AposModalTabs.stories.js:85]: This is inaccessible, fwiw. Not necessary in this story.

File: src/components/modalTabs/AposModalTabs.vue [hunk 2]
```diff
@@ -3,52 +3,26 @@
...
-      uid: (Math.floor(Math.random() * Math.floor(10000))),
-      current: 0
-    }
-  },
   props: {
-    groups: Array
-  },
-  mounted: function () {
-    this.current = this.uid;
+    groups: Array,
```
Comment (abea) [inline src/components/modalTabs/AposModalTabs.vue:18]: Yeah, this seems important to require, right?

File: src/components/modalTabs/AposModalTabs.stories.js (hunk 1 above)
Comment (stuartromanek) [reply src/components/modalTabs/AposModalTabs.stories.js]: gawn

File: src/components/modalTabs/AposModalTabs.vue (hunk 2 above)
Comment (stuartromanek) [reply src/components/modalTabs/AposModalTabs.vue]: required

**Comment by stuartromanek** [comment]
also set tabs and tabsBody to default to the first item in `groups`'s uid if nothing passed

**Pull request merged**